src/script/cbottoken.cpp

```diff
@@ -210,6 +214,9 @@ std::string GetHelpFilename(ObjectType type)
     if ( type == OBJECT_MOBILErs    )  helpfile = "object/botshld";
     if ( type == OBJECT_MOBILEsa    )  helpfile = "object/botsub";
     if ( type == OBJECT_MOBILEwt    )  helpfile = "object/bottr";
+    if ( type == OBJECT_MOBILEtt    )  helpfile = "object/bottr";
+    if ( type == OBJECT_MOBILEft    )  helpfile = "object/bottr";
+    if ( type == OBJECT_MOBILEit    )  helpfile = "object/bottr";
```

Comment:
Missing a help filename for OBJECT_MOBILEpr

Reply:
There is no way anyone could access such helpfile, as this is not really an object.

Reply:
Type PracticeBot in the editor and click on the name on status bar - I'm pretty sure that opens files from this list

Reply:
Ah, yes, of course. Forgot about that.